CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 #Changelog
 
+## 1.0.16
+### Features Added
+- No code changes from 1.0.15
```

Comment:
Seems unnecessary to mention there are no code changes

Reply:
👍 I'll remove these lines.